### PR TITLE
Second set of updates to overview based on recommendations from…

### PR DESCRIPTION
**Pull request by sg00dwin**
… ux doc https://docs.google.com/a/redhat.com/document/d/1eYqc-EpWU_tJveOw2V-We1cMdf6W3tBIZCAy4sKrv4E/edit?usp=sharing

Overview effected elements
pod-template - component-labels and section-label
pod-mini-donut and text
pipelines

### CODE REVIEW

**Comment by rhamilto** [comment]
I haven't looked at code changes yet, but based on screenshots alone, I have some comments/questions.

I think the spacing between "CONTAINER" and "cakephp-example" could be tightened up a bit.  

Did you test the other instances where the pod-template directive is used?

Is it possible to fix the vertical alignment of the mini donut and the kebab (existing bug)?  They are misaligned between 768px and 992px.

Can the spacing between the donut and "pod(s)" be decreased to what they were originally?  It's currently a bit hard to tell they're related.

On a related note, the decrease in the gap between the mini donut and the kebab makes those two items flow together and seem related.  Is it possible to restore the old spacing?

**Comment by sg00dwin** [comment]
@rhamilto pr updated

**New above old**
screen shot 2018-01-08 at 5 05 50 pm



The new 'CONTAINER' `component-label.section-label` in the pod-template is only shown on the overview.  screenshot below shows why.

**Old pod-template within details**
***
screen shot 2018-01-08 at 4 16 28 pm

**New pod-template within details** 
***
screen shot 2018-01-08 at 4 16 13 pm

**What the overview pod-template display looks like in the detail page view**
***
screen shot 2018-01-08 at 4 36 40 pm

File: app/styles/_core.less [hunk 1]
```diff
@@ -116,7 +116,8 @@
     border-bottom: 1px solid #eee;
     padding-bottom: 10px;
   }
-  overlay-panel h3 {
+  overlay-panel h3,
```
Comment (rhamilto) [inline app/styles/_core.less]: Nit:  nest the h3 so you don't have to repeat it?

File: app/styles/_overview.less [hunk 2]
```diff
@@ -239,7 +222,8 @@
   }
   .list-pf-actions {
     // if no actions-dropdown-kebab is present, still take up space
-    min-width: 19px;
+    min-width: 35px;
+    justify-content: flex-end;
```
Comment (rhamilto) [inline app/styles/_overview.less]: Nit: alpha

File: app/styles/_overview.less [hunk 3]
```diff
@@ -191,6 +179,9 @@
   }
   .expanded-section {
     margin-top: 20px;
+    &.no-margin {
+    margin: 0; // when expanded-section is first within list-pf-container
```
Comment (rhamilto) [inline app/styles/_overview.less]: Nit:  indentation

File: app/views/overview/_build-counts.html [hunk 4]
```diff
@@ -1,25 +1,25 @@
...
-        <span ng-switch-when="Failed" class="status-icon">
-          <i class="pficon pficon-error-circle-o text-danger"></i>
-        </span>
-        <span ng-switch-default>
-          <status-icon status="phase"></status-icon>
-        </span>
-      </span>
-      {{buildCounts.countByPhase[phase]}}
-      <span class="sr-only">{{phase}}</span>
+  ng-if="buildCounts.countByPhase[phase]"
```
Comment (rhamilto) [inline app/views/overview/_build-counts.html]: Nit:  unnecessary whitespace changes?

File: app/views/overview/_build-counts.html [hunk 5]
```diff
@@ -1,25 +1,25 @@
...
+    <span ng-switch-when="Failed" class="status-icon">
+      <i class="pficon pficon-error-circle-o text-danger"></i>
     </span>
+    <span ng-switch-default>
+      <status-icon status="phase"></status-icon>
+    </span>
+  </span>
+  {{buildCounts.countByPhase[phase]}}
+  <span class="sr-only">{{phase}}</span>
+</span>
```
Comment (rhamilto) [inline app/views/overview/_build-counts.html]: Nit:  indentation

File: app/views/overview/_networking.html [hunk 6]
```diff
@@ -1,10 +1,9 @@
 <div ng-if="networking.rowServices | size" class="expanded-section networking-section">
-  <div class="section-title hidden-xs">Networking</div>
+  <div class="component-label section-label hidden-xs">Networking</div>
   <div ng-repeat="service in networking.rowServices" class="row">
     <div class="col-sm-5 col-md-6">
-      <div class="component-label">
-        Service
-        <span class="sublabel">Internal Traffic</span>
+      <div class="resource-label">
+        Service -Internal Traffic
```
Comment (rhamilto) [inline app/views/overview/_networking.html]: Missing space after hyphen

**Comment by rhamilto** [comment]
Pipeline details used to be centered in the cell.  

screen shot 2018-01-09 at 12 04 50 pm

Did you intentionally change it?

**Comment by sg00dwin** [comment]
In the design doc she specified the pipeline summary be left aligned. 
https://docs.google.com/document/d/1eYqc-EpWU_tJveOw2V-We1cMdf6W3tBIZCAy4sKrv4E/edit#bookmark=id.v4aowpn0ca1j

**Comment by sg00dwin** [comment]
@rhamilto updated

**Comment by jwforres** [comment]
The two horizontal rules created underneath Template and CONTAINER seems wrong to me (the pod template on detail page screenshot).

**Comment by jwforres** [comment]
the number is not centered well in the mini-donut

**Comment by sg00dwin** [comment]
@jwforres  "The two horizontal rules created underneath Template and CONTAINER seems wrong to me (the pod template on detail page screenshot)."

That screenshot was just to show the specific issue "two horizontal rules" as it would appear, IF we reuse the exact pod-template presentation requested by UX, from the overview. 

So after speaking with @rhamilto we came up with the following.
Here's how ` <pod-template>` is displayed when it's under a `<h3>Template</h3>` heading.

screen shot 2018-01-09 at 4 18 00 pm

**Comment by sg00dwin** [comment]
Tweaked the text centering in `pod-donut-mini`. better?

screen shot 2018-01-09 at 6 10 07 pm

**Comment by jwforres** [comment]
Yep much better

**Comment by sg00dwin** [comment]
**Before**
screen shot 2018-01-10 at 3 58 34 pm

**After**
screen shot 2018-01-10 at 3 36 23 pm

File: app/styles/_components.less [hunk 7]
```diff
@@ -57,13 +79,22 @@ code.command {
...
+.pods {
+  display: inline-block;
+  margin-right: 25px;
+}
+pod-donut[mini] {
+  align-items: center;
+  display: flex;
+  position: relative;
+}
+// align with right outer edge of 'View events | Cancel'
```
Comment (rhamilto) [inline app/styles/_components.less]: It's not clear to me what this comment applies to.

File: app/styles/_list-pf.less [hunk 8]
```diff
@@ -28,7 +28,6 @@
   .dropdown-kebab-pf {
     .dropdown-toggle{
       font-size: 18px;
-      margin-right: -5px;
```
Comment (rhamilto) [inline app/styles/_list-pf.less]: This change results in the dropdown caret and the kebab being misaligned.

screen shot 2018-01-11 at 11 15 45 am

File: app/views/_pod-template-container.html [hunk 9]
```diff
@@ -8,8 +8,10 @@
  -->
 
 <div class="pod-template">
-  <div class="component-label"><span ng-bind-template="{{labelPrefix||'Container'}}:"></span> {{container.name}}</div>
-
+  <div ng-if="!detailed">
```
Comment (rhamilto) [inline app/views/_pod-template-container.html]: Could keep the markup simpler by applying the ng-if to div.component-label.section-label rathern than adding a new div.

File: app/styles/_core.less [hunk 10]
```diff
@@ -1099,6 +1101,7 @@ status-icon {
   // so that status-icons have a fixed width (like .fa-fw, but better sized)
   // this ensures when the icons change, the text that follows doesn't shift
   .fa, .pficon {
+    font-size: 14px;
```
Comment (rhamilto) [inline app/styles/_core.less]: I think we may want to scope this change to only the unexpanded rows as I'm noticing the status-icons seem a little wonky in other contexts now (like the expanded row where builds appear).

**Comment by rhamilto** [comment]
Pipeline build time can wrap to two lines.  
screen shot 2018-01-11 at 11 42 08 am

**Comment by jennyhaines** [comment]
@rhamilto - keeping it centered is fine with me since it takes care of the text size issue! Thanks.

**Comment by rhamilto** [comment]
Spacing after "PIPELINES" is less than the other .component-label.

screen shot 2018-01-11 at 2 58 36 pm
screen shot 2018-01-11 at 2 59 30 pm

File: app/styles/_list-pf.less (hunk 8 above)
Comment (sg00dwin) [reply app/styles/_list-pf.less]: Corrected alignment
screen shot 2018-01-11 at 3 30 51 pm

**Comment by sg00dwin** [comment]
@spadgett 
I've pushed the latest updates that address @rhamilto's comments.

**Comment by sg00dwin** [comment]
**IE 11 & 16 screenshots**

screen shot 2018-01-12 at 2 00 34 pm

screen shot 2018-01-12 at 2 03 37 pm

**Monitoring page**
screen shot 2018-01-12 at 2 41 40 pm

**Comment by sg00dwin** [comment]
@spadgett PTAL

**Comment by jwforres** [comment]
@sg00dwin for some reason jenkins is failing on your dist even though travis passed

**Comment by jwforres** [comment]
Ah i think this might be from the patternfly update.  @sg00dwin can you rebase on master, install deps, and rebuild your dist

**Comment by jwforres** [comment]
FWIW when i kicked Travis again it failed now
